docs/create-packages/creating-a-package-dotnet-cli.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Create a NuGet package using the dotnet CLI
+description: A detailed guide to the process of designing and creating a NuGet package, including key decision points like files and versioning.
+author: karann-msft
+ms.author: karann
+ms.date: 07/09/2019
+ms.topic: conceptual
+---
+
+# Create a NuGet package using the dotnet CLI
+
+No matter what your package does or what code it contains, you use one of the CLI tools, either `nuget.exe` or `dotnet.exe`, to package that functionality into a component that can be shared with and used by any number of other developers. This article describes how to create a package using the dotnet CLI. To install the `dotnet` CLI, see [Install NuGet client tools](../install-nuget-client-tools.md). Starting in Visual Studio 2017, the dotnet CLI is included with .NET Core workloads.
+
+For .NET Core and .NET Standard projects that use the [SDK-style format](../resources/check-project-format.md), and any other SDK-style projects, NuGet uses information in the project file directly to create a package. For detailed steps, see [Create .NET Standard Packages with dotnet CLI](../quickstart/create-and-publish-a-package-using-the-dotnet-cli.md), [Create .NET Standard Packages with Visual Studio](../quickstart/create-and-publish-a-package-using-visual-studio.md) or [NuGet pack and restore as MSBuild targets](../reference/msbuild-targets.md).
+
+Technically speaking, a NuGet package is just a ZIP file that's been renamed with the `.nupkg` extension and whose contents match certain conventions. This topic describes the detailed process of creating a package that meets those conventions.
```

Comment:
This might be interesting on a generic page introducing NuGet, but I'm not sure it's relevant on a page specifically about creating pages with the dotnet cli.

Reply:
I can remove some of this info